webpack.config.js

```diff
@@ -0,0 +1,63 @@
+const webpack = require('webpack')
+const path = require('path')
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin')
+module.exports = {
+  // entry: './packages/client-browser/src/index.ts',
+  target: 'web',
+  stats: 'errors-only',
+  devtool: 'eval-source-map',
+  node: {
+    global: true,
+    __filename: true,
+    __dirname: true,
+  },
+  module: {
+    rules: [
+      {
+        test: /\.ts$/,
+        use: 'ts-loader',
+        exclude: [/node_modules/, /\*\*\/client-node/],
+      },
+    ],
+  },
+  output: {
+    path: path.resolve(__dirname, './webpack'),
+    // filename: 'browser.js',
+    libraryTarget: 'umd',
+    globalObject: 'this',
+    libraryExport: 'default',
+    umdNamedDefine: true,
+    library: 'clickhouse-js',
+  },
+  resolve: {
+    extensions: [
+      '.ts',
+      '.js', // for 3rd party modules in node_modules
+    ],
+    plugins: [
+      new TsconfigPathsPlugin({
+        configFile: 'tsconfig.dev.json',
+        logLevel: 'ERROR',
+      }),
+    ],
+    fallback: {
+      buffer: false,
```

Comment:
That's a weird behavior of tests - Webpack still wants to assemble the rest of the Node.js code (I think because of how I am creating the test client) and complaints about incompatibilities. These warnings are muted when all polyfills are explicitly disabled - it seems it finally just skips the code now.